client/.env.template

```diff
@@ -0,0 +1,3 @@
+REACT_APP_INFURA_URI=
+REACT_APP_CHAIN_ID=
+REACT_APP_BANK_CONTRACT_ADDRESS=
```

Comment:
I don't think this gets used, bank is hard coded to 0x4Aba792D41928D4bC65A4b77d840Fc6418940438